src/Config.h

```diff
@@ -133,6 +135,20 @@ class DOCKS_EXPORT Config
     void setQmlEngine(QQmlEngine *);
     QQmlEngine* qmlEngine() const;
 
+    ///@brief Set the minimal size of the docks
```

Comment:
maybe use the `/**` doxygen style for consistency with the rest ?  see src/private/multisplitter/AnchorGroup_p.h for instance

Reply:
I can, but on the lines above, this style is used